pallets/avn-finality-tracker/src/mock.rs

```diff
@@ -73,12 +73,21 @@ impl system::Config for TestRuntime {
     type MaxConsumers = frame_support::traits::ConstU32<16>;
 }
 
+impl pallet_timestamp::Config for TestRuntime {
```

Comment:
To avoid adding `pallet_timestamp` as a dependency everywhere and implement it in mock.rs, you can implement `UnixTime` in avn and use that in your mock.rs files. So something like this should work:

- In avn, lib.rs
```
pub struct TestUnixTime {
}
impl UnixTime for TestUnixTime  {
    fn now() -> core::time::Duration {
      core::time::Duration::from_millis(0u64)
    }
}
```

- In your mock.rs files:
```
type TimeProvider = pallet_avn::TestUnixTime;
```